### PR TITLE
Allow aggregate stages with same name

### PR DESCRIPTION
**Pull request by dplewis**
Instead of converting array to json for the payload just pass it though.

Since I can't pass an array to the RESTController, The pipeline key needs to be added to the server for this to work.

### CODE REVIEW

File: integration/test/ParseQueryAggregateTest.js [hunk 1]
```diff
@@ -55,6 +55,48 @@ describe('Parse Aggregate Query', () => {
...
+      },
+    }, {
+      // lookup returns an array, Deconstructs an array field to objects
+      unwind: {
+        path: '$tempPointer',
+      },
+    }, {
+      match: { 'tempPointer.value': 2 },
+    }
+  ];
```
[the remark this answers is not in the document]
Comment (flovilmart) [reply integration/test/ParseQueryAggregateTest.js]: Fix it before we merge please...

**Comment by dplewis** [comment]
@flovilmart I'm waiting for this to be merged. https://github.com/parse-community/parse-server/pull/4959

**Comment by flovilmart** [comment]
Yeah but please fix the code first.

**Comment by flovilmart** [comment]
Doing the eslint on this repo!

**Comment by dplewis** [comment]
That would be great

**Pull request merged**